### PR TITLE
Fix markdown table formatting

### LINKED ISSUE #23385: Fix .md table formatting in about-github-connect.md
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

[about-github-connect.md](https://github.com/github/docs/blob/main/content/admin/configuration/configuring-github-connect/about-github-connect.md) 

### What changes are you suggesting?

Fix markdown table formatting that currently exist in [about-github-connect.md](https://github.com/github/docs/blob/main/content/admin/configuration/configuring-github-connect/about-github-connect.md) 

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by ieshreya**
### Why:

Closes #23385

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by cmwilson21** [comment]
@ieshreya Thanks for submitting a PR! I edited your post slightly so it connects to your issue 👍 

I'll get this triaged for review! :zap:

File: content/admin/configuration/configuring-github-connect/about-github-connect.md [hunk 1]
```diff
@@ -27,9 +27,9 @@ After enabling {% data variables.product.prodname_github_connect %}, you will be
...
 
-Feature | Description | More information |
-------- | ----------- | ---------------- |{% ifversion ghes %}
-Automatic user license sync | Manage license usage across your {% data variables.product.prodname_enterprise %} deployments by automatically syncing user licenses from {% data variables.location.product_location %} to {% data variables.product.prodname_ghe_cloud %}. | "[Enabling automatic user license sync for your enterprise](/admin/configuration/configuring-github-connect/enabling-automatic-user-license-sync-for-your-enterprise)"{% endif %}{% ifversion ghes or ghae %}
+|Feature | Description | More information |
+| ----------- | ----------- | ----------- |
+{% ifversion ghes %} Automatic user license sync | Manage license usage across your {% data variables.product.prodname_enterprise %} deployments by automatically syncing user licenses from {% data variables.location.product_location %} to {% data variables.product.prodname_ghe_cloud %}. | "[Enabling automatic user license sync for your enterprise](/admin/configuration/configuring-github-connect/enabling-automatic-user-license-sync-for-your-enterprise)"{% endif %}{% ifversion ghes or ghae %}
```
Comment (mchammer01) [inline content/admin/configuration/configuring-github-connect/about-github-connect.md:32]: Moving this Liquid versioning to the next line as it's breaking the formatting for GHAE.
```suggestion
{% ifversion ghes %} Automatic user license sync | Manage license usage across your {% data variables.product.prodname_enterprise %} deployments by automatically syncing user licenses from {% data variables.location.product_location %} to {% data variables.product.prodname_ghe_cloud %}. | "[Enabling automatic user license sync for your enterprise](/admin/configuration/configuring-github-connect/enabling-automatic-user-license-sync-for-your-enterprise)"{% endif %}
```

File: content/admin/configuration/configuring-github-connect/about-github-connect.md [hunk 2]
```diff
@@ -27,9 +27,9 @@ After enabling {% data variables.product.prodname_github_connect %}, you will be
...
 {% data variables.product.prodname_dependabot %} | Allow users to find and fix vulnerabilities in code dependencies. | "[Enabling {% data variables.product.prodname_dependabot %} for your enterprise](/admin/configuration/configuring-github-connect/enabling-dependabot-for-your-enterprise)"{% endif %}
```
Comment (mchammer01) [inline content/admin/configuration/configuring-github-connect/about-github-connect.md:33]: Moved the versioning to here, that'll fix the issue with the GHAE version of the article
```suggestion
{% ifversion ghes or ghae %}{% data variables.product.prodname_dependabot %} | Allow users to find and fix vulnerabilities in code dependencies. | "[Enabling {% data variables.product.prodname_dependabot %} for your enterprise](/admin/configuration/configuring-github-connect/enabling-dependabot-for-your-enterprise)"{% endif %}
```

File: content/admin/configuration/configuring-github-connect/about-github-connect.md (hunk 1 above)
Comment (mchammer01) [reply content/admin/configuration/configuring-github-connect/about-github-connect.md:32]: See this screenshot: 
ghae-version

File: content/admin/configuration/configuring-github-connect/about-github-connect.md [hunk 3]
```diff
@@ -63,15 +63,15 @@ When you enable {% data variables.product.prodname_github_connect %} or specific
...
 
-Feature | Data | Which way does the data flow? | Where is the data used? | 
-------- | ---- | --------- | ------ |{% ifversion ghes %}
-Automatic user license sync | Each {% data variables.product.product_name %} user's user ID and email addresses | From {% data variables.product.product_name %} to {% data variables.product.prodname_ghe_cloud %} | {% data variables.product.prodname_ghe_cloud %} |{% endif %}{% ifversion ghes or ghae %}
+| Feature | Data | Which way does the data flow? | Where is the data used? | 
+| ------- | ---- | --------- | ------ |
+{% ifversion ghes %} Automatic user license sync | Each {% data variables.product.product_name %} user's user ID and email addresses | From {% data variables.product.product_name %} to {% data variables.product.prodname_ghe_cloud %} | {% data variables.product.prodname_ghe_cloud %} |{% endif %}{% ifversion ghes or ghae %}
 {% data variables.product.prodname_dependabot_alerts %} | Vulnerability alerts | From {% data variables.product.prodname_dotcom_the_website %} to {% data variables.product.product_name %} | {% data variables.product.product_name %} |{% endif %}{% ifversion dependabot-updates-github-connect %}
```
Comment (mchammer01) [inline content/admin/configuration/configuring-github-connect/about-github-connect.md:75]:
```suggestion
{% ifversion ghes or ghae %}{% data variables.product.prodname_dependabot_alerts %} | Vulnerability alerts | From {% data variables.product.prodname_dotcom_the_website %} to {% data variables.product.product_name %} | {% data variables.product.product_name %} |{% endif %}{% ifversion dependabot-updates-github-connect %}
```

**Review by mchammer01 — COMMENTED** [review]
@ieshreya 👋🏻 - thanks for improving our docs 💖 
I made a few minor suggestions to fix the formatting on a different version of the article.
Once I apply those, I'll check that the tables look fine in all versions, and will hopefully merge this 🙇🏻

**Comment by mchammer01** [comment]
Hmm, I am sorry, the formatting issue is still there after I've applied my suggestions. Investigating. Thanks for your patience 🙇🏻 🙂

File: content/admin/configuration/configuring-github-connect/about-github-connect.md [hunk 4]
```diff
@@ -27,10 +27,10 @@ After enabling {% data variables.product.prodname_github_connect %}, you will be
...
 
-Feature | Description | More information |
-------- | ----------- | ---------------- |{% ifversion ghes %}
-Automatic user license sync | Manage license usage across your {% data variables.product.prodname_enterprise %} deployments by automatically syncing user licenses from {% data variables.location.product_location %} to {% data variables.product.prodname_ghe_cloud %}. | "[Enabling automatic user license sync for your enterprise](/admin/configuration/configuring-github-connect/enabling-automatic-user-license-sync-for-your-enterprise)"{% endif %}{% ifversion ghes or ghae %}
-{% data variables.product.prodname_dependabot %} | Allow users to find and fix vulnerabilities in code dependencies. | "[Enabling {% data variables.product.prodname_dependabot %} for your enterprise](/admin/configuration/configuring-github-connect/enabling-dependabot-for-your-enterprise)"{% endif %}
+|Feature | Description | More information |
+| ----------- | ----------- | ----------- |
```
Comment (mchammer01) [inline content/admin/configuration/configuring-github-connect/about-github-connect.md:31]: Revert my change
```suggestion
| ----------- | ----------- | ----------- |{% ifversion ghes %}
```

File: content/admin/configuration/configuring-github-connect/about-github-connect.md [hunk 5]
```diff
@@ -27,10 +27,10 @@ After enabling {% data variables.product.prodname_github_connect %}, you will be
...
+{% ifversion ghes %} Automatic user license sync | Manage license usage across your {% data variables.product.prodname_enterprise %} deployments by automatically syncing user licenses from {% data variables.location.product_location %} to {% data variables.product.prodname_ghe_cloud %}. | "[Enabling automatic user license sync for your enterprise](/admin/configuration/configuring-github-connect/enabling-automatic-user-license-sync-for-your-enterprise)"{% endif %}
+{% ifversion ghes or ghae %}{% data variables.product.prodname_dependabot %} | Allow users to find and fix vulnerabilities in code dependencies. | "[Enabling {% data variables.product.prodname_dependabot %} for your enterprise](/admin/configuration/configuring-github-connect/enabling-dependabot-for-your-enterprise)"{% endif %}
```
Comment (mchammer01) [inline content/admin/configuration/configuring-github-connect/about-github-connect.md:33]:
```suggestion
{% data variables.product.prodname_dependabot %} | Allow users to find and fix vulnerabilities in code dependencies. | "[Enabling {% data variables.product.prodname_dependabot %} for your enterprise](/admin/configuration/configuring-github-connect/enabling-dependabot-for-your-enterprise)"{% endif %}
```

File: content/admin/configuration/configuring-github-connect/about-github-connect.md [hunk 6]
```diff
@@ -27,10 +27,10 @@ After enabling {% data variables.product.prodname_github_connect %}, you will be
 
 After you configure the connection between {% data variables.location.product_location %} and {% data variables.product.prodname_ghe_cloud %}, you can enable individual features of {% data variables.product.prodname_github_connect %} for your enterprise.
 
-Feature | Description | More information |
-------- | ----------- | ---------------- |{% ifversion ghes %}
-Automatic user license sync | Manage license usage across your {% data variables.product.prodname_enterprise %} deployments by automatically syncing user licenses from {% data variables.location.product_location %} to {% data variables.product.prodname_ghe_cloud %}. | "[Enabling automatic user license sync for your enterprise](/admin/configuration/configuring-github-connect/enabling-automatic-user-license-sync-for-your-enterprise)"{% endif %}{% ifversion ghes or ghae %}
-{% data variables.product.prodname_dependabot %} | Allow users to find and fix vulnerabilities in code dependencies. | "[Enabling {% data variables.product.prodname_dependabot %} for your enterprise](/admin/configuration/configuring-github-connect/enabling-dependabot-for-your-enterprise)"{% endif %}
+|Feature | Description | More information |
```
Comment (mchammer01) [inline content/admin/configuration/configuring-github-connect/about-github-connect.md:30]:
```suggestion
| Feature | Description | More information |
```

File: content/admin/configuration/configuring-github-connect/about-github-connect.md [hunk 7]
```diff
@@ -63,16 +63,16 @@ When you enable {% data variables.product.prodname_github_connect %} or specific
...
 ### Data transmitted by individual features of {% data variables.product.prodname_github_connect %}
 
 Additional data is transmitted if you enable individual features of {% data variables.product.prodname_github_connect %}.
 
-Feature | Data | Which way does the data flow? | Where is the data used? | 
-------- | ---- | --------- | ------ |{% ifversion ghes %}
-Automatic user license sync | Each {% data variables.product.product_name %} user's user ID and email addresses | From {% data variables.product.product_name %} to {% data variables.product.prodname_ghe_cloud %} | {% data variables.product.prodname_ghe_cloud %} |{% endif %}{% ifversion ghes or ghae %}
-{% data variables.product.prodname_dependabot_alerts %} | Vulnerability alerts | From {% data variables.product.prodname_dotcom_the_website %} to {% data variables.product.product_name %} | {% data variables.product.product_name %} |{% endif %}{% ifversion dependabot-updates-github-connect %}
+| Feature | Data | Which way does the data flow? | Where is the data used? | 
+| ------- | ---- | --------- | ------ |
```
Comment (mchammer01) [inline content/admin/configuration/configuring-github-connect/about-github-connect.md:73]:
```suggestion
| ------- | ---- | --------- | ------ |{% ifversion ghes %}
```

**Review by mchammer01 — COMMENTED** [review]
Many apologies, I think that reverting my changes for the Liquid versioning should work (I've left your other updates alone 😅) 🤞🏻

**Review by mchammer01 — APPROVED** [review]
LGTM ✨ 
And thanks for your patience with me 💖

**Comment by ieshreya** [comment]
Thanks a lot for the consistent effort, @cmwilson21 and @mchammer01! 🙇🏻❤️

**Comment by mchammer01** [comment]
@ieshreya 👋🏻 - Thank YOU for helping us improve our docs ✨ ❤️

**Pull request merged**